mbed_host_tests/host_tests_conn_proxy/conn_proxy.py

```diff
@@ -130,6 +135,7 @@ def __notify_conn_lost():
     sync_behavior = int(config.get('sync_behavior', 1))
     sync_timeout = config.get('sync_timeout', 1.0)
     conn_resource = config.get('conn_resource', 'serial')
+    last_sync = 0
```

Comment:
Nit pick: Could you use `True` and `False` vs `0` and `1`? This is more consistent with the rest of the codebase.